### PR TITLE
Initial implementation.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
This PR updates the existing `ed25519-verification-key-2018` library to use the new 2020 spec.

~~Currently, only uses the isomorphic `@stablelib/ed25519` lib for crypto. If we need to, we can add a separate PR to add native Node.js crypto.~~
(Native crypto now added.)

### CODE REVIEW

**Comment by dlongley** [comment]
> If we need to, we can add a separate PR to add native Node.js crypto.

Yes, we'll need to do that.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - name: Run eslint
+      run: npm run lint
+#  coverage:
```
Comment (davidlehn) [inline .github/workflows/main.yml:52]: `nyc` is being installed, should probably add the coverage commands to use it and enable coverage here.

Comment (dmitrizagidulin) [reply .github/workflows/main.yml:52]: @davidlehn let's open a separate issue/PR for that.

**Review by dlongley — APPROVED** [review]
Approving for my part here, over to @davidlehn for any remaining webpack/build issues.

**Comment by dmitrizagidulin** [comment]
Merging since approved; will fine-tune webpack issues in separate PR. (Preliminary downstream testing with vc-js suggests that the webpack works.)

**Pull request merged**